data/categories.txt

```diff
@@ -6271,7 +6275,7 @@ nl:^Droogkuis|wassen
 fi:^Kuivapesula|pesula
 fr:^Nettoyage à sec|Lavage
 de:^Chemische Reinigung|Reinigung
-hu:^Ruhatisztító|tisztító
+hu:^Ruhatisztító|vegytisztító|2patyolat|2tisztító
```

Comment:
Are these important enough to start suggesting them after the first 2 letters?

Reply:
​Well, it should be rather 3 or 4 there.​